examples/next/pages/index.js

```diff
@@ -13,8 +13,8 @@ export default class Page extends React.Component {
       return { cache };
     }
 
-    // get cache from window
-    return { cache: window.__NEXT_DATA__.props.pageProps };
+    // get cache from props
+    return { cache: this.props.cache };
```

Comment:
I don't think we need to get cache specifically from `window.__NEXT_DATA__`. NextJs seems to already insert all props in `window.__NEXT_DATA__` into props for the page in the client.

Reply:
I just realized that this method is a static method so why can we read `this` inside? 🤔

Reply:
I think I might have made a mistake, let me check again.